pkgs/development/tools/ytt/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "ytt";
+  version = "0.30.0";
+
+  src = fetchFromGitHub {
+    owner = "vmware-tanzu";
+    repo = "carvel-ytt";
+    rev = "v${version}";
+    sha256 = "0v9wp15aj4r7wif8i897zwj3c6bg41b95kk7vi3a3bzin814qn6l";
+  };
+
+  goPackagePath = "github.com/vmware-tanzu/carvel-ytt";
+
+  vendorSha256 = null;
+
```

Comment:
```suggestion

subPackages = [ "cmd/ytt" ];

```

Reply:
I'll do this, but there are no tests ran since they're tucked away in other subpackages. It does make for a smaller file overall.